sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -230,3 +238,49 @@ def _feast_to_milvus_data_type(self, feast_type: FeastType) -> DataType:
             # TODO: Need to think about list of binaries and list of bytes
             # FeastType.BYTES_LIST: DataType.BINARY_VECTOR
         }.get(feast_type, None)
+
+    def _format_data_for_milvus(self, feast_data):
+        """
+        Data stored into Milvus takes the grouped representation approach where each feature value is grouped together:
+        [[1,2], [1,3]], [John, Lucy], [3,4]]
+
+        Parameters:
+        feast_data: List[
+            Tuple[EntityKeyProto, Dict[str, ValueProto], datetime, Optional[datetime]]: Data represented for batch write in Feast
+
+        Returns:
+        List[List]: transformed_data: Data that can be directly written into Milvus
+        """
+
+        milvus_data = []
+        for entity_key, values, timestamp, created_ts in feast_data:
+            feature = []
+            for feature_name, val in values.items():
+                val_type = val.WhichOneof("val")
+                if val_type == "float_list_val":
+                    float_list = val.float_list_val.val
+                    final_float_list = np.array(float_list)
+                    feature.append(final_float_list)
+                # TODO: Test out binary lists
+                if val_type == "biinary_list_val":
```

Comment:
typo: should be binary_list_val

Reply:
I think we could simplify this:
`                val_type = val.WhichOneof("val")
                value = getattr(val, val_type).val
                if val_type == "float_list_val":
                    value = np.array(value)
                feature.append(value)`

what do you think?

Reply:
This only works for floats

value = getattr(val, val_type).val

else 

for non floats
value = getattr(val, val_type)

Maybe I am not understanding it correctly but I don't feel like this approach works